cloudlift/config/environment_configuration.py

```diff
@@ -384,8 +381,7 @@ def _validate_changes(self, configuration):
         except ValidationError as validation_error:
             error_path = str(".".join(list(validation_error.relative_path)))
             if error_path:
-                log_err(validation_error.message + " in " + error_path)
+                raise UnrecoverableException(validation_error.message + " in " + error_path)
             else:
-                log_err(validation_error.message)
-            exit(0)
```

Comment:
Previously we exit with `0` for validation errors. I've made it consistent and simpler to return with exit `1`.